src/app/state/staking/selectors.ts

```diff
@@ -6,12 +6,19 @@ import { initialState } from '.'
 const selectSlice = (state: RootState) => state.staking || initialState
 
 export const selectStaking = createSelector([selectSlice], state => state)
-export const selectValidators = createSelector([selectStaking], state => state.validators.list)
-export const selectValidatorsTimestamp = createSelector([selectStaking], state => state.validators.timestamp)
+export const selectValidators = createSelector([selectStaking], state => state.validators?.list ?? [])
+export const selectValidatorsTimestamp = createSelector(
+  [selectStaking],
+  state => state.validators?.timestamp ?? 0,
```

Comment:
When it can be undefined? Do we want to show 1970 in updateValidatorsError?

Reply:
1970 should never appear with current code. It is undefined until first `maybeRefreshValidators` (initialState = { validators: null })

Annoying options:
1. keep default 0, so type is number
2. default to null, and force checking nullable everywhere we want to display timestamp. You have to check nullable even if we already check updateValidatorsError.
3. create selector for `state.validators`, so that it is a single variable for nullable checks. But it probably isn't best practice, not minimal for mocking, and redux wouldn't detect changes if subfield changed.

Reply:
Hm, wait, 2. is fine if we just use non-null assertion